commcare_connect/opportunity/export.py

```diff
@@ -103,10 +103,8 @@ def export_empty_payment_table(opportunity: Opportunity) -> Dataset:
         "Username",
         "Phone Number",
         "Name",
-        "Payment Accrued",
-        "Payment Completed",
         "Payment Amount",
-        "Paid date",
+        "Payment Date (YYYY-MM-DD)",
     ]
```

Comment:
I think including the format in the header title is a little gross, especially if we have good error messages.